src/VSCodeExtension/src/tests/unit/control-structure.test.ts

```diff
@@ -0,0 +1,98 @@
+import "mocha";
```

Comment:
I'd suggest a unit test to ensure that `if` occuring in the beginning, middle, or end of an identifier name doesn't get replaced.

Reply:
cc: @amrondonp

Reply:
Sure, Will do